src/lib/execPromise.ts

```diff
@@ -37,33 +41,40 @@ export type ExecPromiseArg = {
   printOutput?: boolean | undefined;
 };
 
-export const execPromise = ({
+export const execPromise = async ({
   command,
   args,
   flags,
   options,
   spinnerMessage,
   timeout,
-  printOutput = false,
+  printOutput: printOutputArg = false,
```

Comment:
why `printOutputArg` starts from a lower case? looks strange

and why do you need explicit type here anyway?

Reply:
it's not a type it's a variable

Reply:
I don't understand. What does `printOutput: printOutputArg = false` mean?

Reply:
printOutput - is destructuring; printOutput: printOutputArg - is renaming of the destructured value; printOutput: printOutputArg = false - is a default value in case the destructured value is undefined

Reply:
omg :) ok